packages/terra-clinical-site/package.json

```diff
@@ -29,9 +29,11 @@
   "dependencies": {
     "prop-types": "^15.5.8",
     "react-intl": "^2.3.0",
+    "terra-alert": "^1.0.0",
```

Comment:
Leaving this for now. And logged an issue to investigate further https://github.com/cerner/terra-core/issues/730